controllers/SiteController.php

```diff
@@ -241,4 +240,10 @@ public function actionCommunity()
         $this->sectionTitle = 'Community Resources';
         return $this->render('community');
     }
+
+    public function actionRenderMarkdown()
+    {
+        $markdown = Yii::$app->request->post('content');
+        return Yii::$app->formatter->asGuideMarkdown($markdown);
```

Comment:
we should probably restrict this to logged in users or rate limit it, otherwise one could send arbitrary content for markdown parsing here.

Reply:
Done.